lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+
+    # Translators:
+    context['company_careers_urltext'] = _("Work at {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_contact_urltext'] = _("Contact {company_name}").format(company_name=company_name)
+
+    # Translators:
+    context['company_privacy_urltext'] = _("Privacy Policy")
+
+    # Translators:
+    context['company_tos_urltext'] = _("Terms of Service &amp; Honor Code")
+
+    # Translators:
+    context['document_banner'] = _("{company_name} acknowledges the following student accomplishment").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['logo_subtitle'] = _("Certificate Validation")
+
+    if certificate.mode == 'honor':
+        # Translators:
+        context['certificate_type_description'] = _("An {cert_type} Certificate signifies that an {company_name} "
+                                                    "learner has agreed to abide by {company_name}'s honor code and "
+                                                    "completed all of the required tasks for this course under its "
+                                                    "guidelines.").format(
+            cert_type=context.get('certificate_type'),
+            company_name=company_name
+        )
+    elif certificate.mode == 'verified':
+        # Translators:
+        context['certificate_type_description'] = _("An {cert_type} Certificate signifies that an {company_name} "
+                                                    "learner has agreed to abide by {company_name}'s honor code and "
+                                                    "completed all of the required tasks for this course under its "
+                                                    "guidelines, as well as having their photo ID checked to verify "
+                                                    "their identity.").format(
+            cert_type=context.get('certificate_type'),
+            company_name=company_name
+        )
+    elif certificate.mode == 'xseries':
+        # Translators:
+        context['certificate_type_description'] = _("An {cert_type} Certificate demonstrates a high level of "
+                                                    "achievement in a program of study, and includes verification of "
+                                                    "the student's identity.").format(
+            cert_type=context.get('certificate_type')
+        )
+
+    # Translators:
+    context['copyright_text'] = _('&copy; {year} {company_name}. All rights reserved.').format(
+        year=datetime.now().year,
+        company_name=company_name
+    )
+
+    # Translators:
+    context['document_meta_description'] = _('This is a valid {company_name} certificate for {user_name}, '
+                                             'who participated in {course_org} {course_number}').format(
```

Comment:
If course org, course number should be in that specific order, don't make it two variables. That just increases the odds that translators can mis-translate the strings. YOu can just do `{course}`, and then in the format do `course='{} {}'.format(course.org, course.number)`